allennlp/modules/token_embedders/embedding.py

```diff
@@ -74,9 +74,12 @@ class Embedding(TokenEmbedder, Registrable):
         construct it in the original training. We store vocab_namespace used during the original
         training as an attribute, so that it can be retrieved during fine-tuning.
     pretrained_file : `str`, (optional, default=None)
-        Used to keep track of what is the source of the weights and loading more embeddings at test time.
-        **It does not load the weights from this pretrained_file.** For that purpose, use
-        `Embedding.from_params`.
+        Path to a file of word vectors to intialize the embedding matrix. It can be the
+        path to a local file or an URL of a (cached) remote file. Two formats are supported:
```

Comment:
a URL instead of an URL?  Or is that a British / American difference?

Reply:
oh sure, I just copied this comment from below. I'll update it.